pyproject.toml

```diff
@@ -8,3 +8,7 @@ skip-string-normalization = true
 # see https://github.com/psf/black/issues/1778
 force-exclude = '^(?!/(telegram|examples|tests)/).*\.py$'
 include = '(telegram|examples|tests)/.*\.py$'
+
+[tool.isort]  # make sure that the config values are the same in pre-commit-config
+profile = "black"
+line_length = 99
```

Comment:
is that still necessary even if we tell black to use 99 chars already 🤔 I don't recall that being the case in https://github.com/Bibo-Joshi/led-strip-controller where I first used isort …

Reply:
Nice _plug_. [This](https://pycqa.github.io/isort/docs/configuration/profiles.html#black) page told me that it takes 88 chars for `black` config. Not sure if it fetches our set line length from the black config or it needs to explicitly told.. will check

Reply:
okay it needs to be told the line length otherwise it uses 88

Reply:
shame … but okay :)